bin/yaml/tools.yaml

```
@@ -9,7 +9,7 @@ tools:
     check_exe: bin/cmake --help
     symlink: cmake
     targets:
-      - 3.26.1
+      - 3.28.0-rc2
```

As much as I would like to use the official modules support, and jump on one of the rc, would you rather prepare for the impending release and wait for the official release?

```suggestion
      - 3.28.0
```

---

thanks, yes, this is just a PR to remind me to regularly check the status of 3.28